Documentation/DocuBlocks/Rest/Administration/get_admin_status.md

```diff
@@ -0,0 +1,145 @@
+
+@startDocuBlock get_admin_status
+@brief returns Status information of the server.
+
+@RESTHEADER{GET /_admin/status, Return status information, RestStatusHandler}
+
+@RESTDESCRIPTION
+Returns status information about the server.
+
+@RESTRETURNCODES
+
+@RESTRETURNCODE{200}
+Status information was returned successfully.
+
+@RESTREPLYBODY{server,string,required,}
+Always `"arango"`.
+
+@RESTREPLYBODY{license,string,required,}
+ArangoDB Edition, either `"community"` or `"enterprise"`.
+
+@RESTREPLYBODY{version,string,required,}
+The server version as a string.
+
+@RESTREPLYBODY{mode,string,required,}
+Either `"server"` or `"console"`. **Deprecated**, use `operationMode` instead.
+
+@RESTREPLYBODY{operationMode,string,required,}
+Either `"server"` or `"console"`.
+
+@RESTREPLYBODY{foxxApi,boolean,required,}
+Whether the Foxx API is enabled.
+
+@RESTREPLYBODY{host,string,required,}
+A host identifier defined by the `HOST` or `NODE_NAME` environment variable,
+or a fallback value using a machine identifier or the cluster/Agency address.
+
+@RESTREPLYBODY{hostname,string,optional,}
+A hostname defined by the `HOSTNAME` environment variable.
+
+@RESTREPLYBODY{pid,number,required,}
+The process ID of _arangod_.
+
+@RESTREPLYBODY{serverInfo,object,required,get_admin_status_server_info}
+Information about the server status.
+
+@RESTSTRUCT{progress,get_admin_status_server_info,object,required,get_admin_status_server_info_progress}
+Startup and recovery information.
+
+You can check for changes to determine whether progress was made between two
+calls, but you should not rely on specific values as they may change between
+ArangoDB versions. The values are only expected to change during the startup and
+shutdown, i.e. while `maintenance` is `true`.
+
+You need to start _arangod_ with the `--server.early-connections` startup option
+enabled to be able to query the endpoint during the startup phase.
```

Comment:
@jsteemann Maybe this is better?
```suggestion
enabled to be able to query the endpoint during the entire startup process.
```

Reply:
This is not correct, because the endpoint is not open during the **entire startup process*, but only after the HTTP REST endpoint is opened. I don't want to nurture the expectation that the HTTP API is always responsive during the entire startup. Which it is not. It is only responsive after the HTTP API is opened, which is during the startup, but only at a certain point.